ctapipe/reco/preprocessing.py

```diff
@@ -48,6 +56,16 @@ def check_valid_rows(table: Table, warn=True, log=LOG) -> np.ndarray:
     return valid
 
 
+def table_to_X(table: Table, features: list, log):
+    """
+    Extract features as numpy ndarray to be given to sklearn from input table
```

Comment:
docstring needs to be more precise: this function selects all valid features, which means only those where there is no nan ?

Reply:
I think only rows containing nans will be dropped by `check_valid_rows`, so `X` only contains events for which all features are valid. I will update the docstring accordingly.